test/trento/domain/cluster/cluster_test.exs

```diff
@@ -116,25 +116,165 @@ defmodule Trento.ClusterTest do
       )
     end
 
-    test "should return an error if the cluster was not registered yet and a command from a non-DC is received" do
-      assert_error(
-        [],
+    test "should add a host the cluster when the cluster has empty/unknown details" do
```

Comment:
maybe `should add a new host to the cluster and keep the details as empty when a non-DC node message is received`